lxd/db/db_unified.go

```diff
@@ -0,0 +1,51 @@
+//go:build linux && cgo && !agent
+
+package db
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/lxc/lxd/lxd/db/query"
+)
+
+// DB represents access to LXD's global and local databases.
+type DB struct {
+	Node    *Node
+	Cluster *Cluster
+}
+
+// Tx is a convenient alias for sql.Tx.
+type Tx = sql.Tx
```

Comment:
Do we need this? Seems to me it would be clearer to have indications that we use standard sql.Tx instead?

Reply:
Fair enough, it just saves 4 characters :)